tests/testsuite/rustdoc_extern_html.rs

```diff
@@ -49,14 +48,19 @@ fn simple() {
     let p = basic_project();
     p.cargo("doc -v --no-deps -Zrustdoc-map")
         .masquerade_as_nightly_cargo(&["rustdoc-map"])
-        .with_stderr_contains(
-            "[RUNNING] `rustdoc [..]--crate-name foo [..]bar=https://docs.rs/bar/1.0.0/[..]",
-        )
+        .with_stderr_data(str![[r#"
+...
+[RUNNING] `rustdoc [..]--crate-name foo [..]--extern-html-root-url [..]bar=https://docs.rs/bar/1.0.0/[..]`
```

Comment:
Glob redaction is required since its not surrounded by quotes on windows.

https://github.com/rust-lang/cargo/actions/runs/9768726905/job/26966828626#step:11:4295
```
   1      - ...
   2      - [RUNNING] `rustdoc [..]--crate-name foo [..]--extern-html-root-url 'bar=https://docs.rs/bar/1.0.0/' [..]`
[omit...]
        8 + [RUNNING] `rustdoc --edition=2018 --crate-type lib --crate-name foo src/lib.rs -o [ROOT]/foo/target/doc --check-cfg cfg(docsrs) --check-cfg "cfg(feature, values())" --error-format=json --json=diagnostic-rendered-ansi,artifacts,future-incompat -C metadata=a7e7c9482575de28 -L dependency=[ROOT]/foo/target/debug/deps --extern bar=[ROOT]/foo/target/debug/deps/libbar-[HASH].rmeta --extern-html-root-url bar=https://docs.rs/bar/1.0.0/ -Zunstable-options --crate-version 0.1.0`
```